bin/snakePipes

```diff
@@ -17,19 +17,36 @@ def parse_arguments():
     subparsers = parser.add_subparsers(title='Commands',
                                        dest='command')
 
-    infoParser = subparsers.add_parser('info', help='Print the location of the various yaml files')
+    infoParser = subparsers.add_parser('info',
+                                      help='Print the location of the various yaml files')
```

Comment:
Your editor likes to offset things by a space.

Reply:
what if I say I did it manually :sweat_smile:

Reply:
:man_facepalming: